CMakeLists.txt

```diff
@@ -17,9 +17,12 @@ else()
     message("-- Configuring release build")
 endif()
 
+# Enable extra warnings to adhere to https://github.com/mapbox/cpp/issues/37
+set(DESIRED_WARNINGS "-Wall -Wextra -Wconversion -Wunreachable-code -Wuninitialized -pedantic-errors -Wold-style-cast -Wno-error=unused-variable -Wshadow -Wfloat-equal")
```

Comment:
`-Weffc++` as well?

Reply:
Nice catch. Fixed in 63c6215. Note: `-Weffc++` does not cause clang++ to error, so is okay to add, but really only does something with g++. This is why in node-cpp-skel it is added to a specific job: https://github.com/mapbox/node-cpp-skel/blob/3f7d2c4bcf3a6162312accbf3554ac17ecbbb407/.travis.yml#L92. But reflecting now I think it would be simplest to inline it in the warnings for all compilers. If you think this makes sense we could modify node-cpp-skel to add the flag to the `binding.gyp` rather than just travis.